### ISSUE TITLE
Use in a Group

### ISSUE THREAD

**Issue by jasonjordan** [avivace/RSSnotifier#11]
Yikes!  I got burned by dropping this into a Group.  It responded to everything everyone was posting!

I note in the source there is a to-do regarding use in groups.

For the moment I've just hacked the msg token so it's broken and the bot stops listening.

Not optimal but I'm not a coder!

Any ETA on getting it to work nicely in a group?

**Comment by avivace** [comment]
Yes, I remember this being a todo. However, IIRC, if you don't set the privacy mode the bot cannot access any message not tagging the bot. Shouldn't this be enough?

I can't test this right now but maybe @dennib can take a look?

**Comment by mrjasonjordan** [comment]
OK, this is where I admit to being a shallow Geek.

Can you explain to me what "privacy mode" is?  If I can get it work as you describe - then yes, it's perfect.

**Comment by mrjasonjordan** [comment]
ok got it. All sorted. All working. Perfect.

Almost. 8-)

I'll add a new issue.

**Comment by mrjasonjordan** [comment]
I'm learning quickly.

Unfortunately, if you have group privacy on, the bot does not have access to messages. 8-(
